app/assets/stylesheets/govuk_admin_template/_bootstrap_colour_overrides.scss

```diff
@@ -5,4 +5,47 @@
   This assumed gray-light was on a light background. We aren't always
   using it in that way. Reset to the original value #999.
 */
+
 $gray-light: lighten(#000, 60%);
+
+/*
+  Updates a bunch of copy that used to be gray to black for things like labels.
+*/
+$text-color: #000;
+
+/*
+  Updates the input to black so it contrasts with the light gray background.
+*/
+$input-color: #000;
+
+/*
+  Updates muted text to gray so it contrasts with the light gray background.
+*/
+$text-muted: #444;
+
+/*
+  Updates the input placeholder text to gray so it contrasts with the light gray background.
+  This is the lightest gray we can use without WAVE complaining.
+*/
+$input-color-placeholder: #666;
```

Comment:
I'm not overly convinced that making this visually accessible is beneficial. It makes the placeholder text colour dangerously close to looking like input. Wdyt?